ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/parameters/FilterParameterTest.java

```diff
@@ -179,6 +180,68 @@ public void jsonWithBlockAndFromAndToParametersIsInvalid() throws Exception {
     assertThat(parsedFilterParameter.isValid()).isFalse();
   }
 
+  @Test
```

Comment:
I'd consider deleting the previous test because the name clash is a bit confusing and the case of `blockHash` doesn't seem significant to the test case.

Reply:
put it into a single test